graders/c/cgrader/cgrader.py

```diff
@@ -415,9 +415,15 @@ def test_run(
         self,
         command: str | Iterable[str],
         input: str | None = None,  # noqa: A002
-        exp_output: str | Iterable[str] | None = None,
+        exp_output: str
+        | re.Pattern[str]
+        | Iterable[str | re.Pattern[str]]
+        | None = None,
         must_match_all_outputs: OutputMatchingOption | bool = "any",  # noqa: FBT001
-        reject_output: str | Iterable[str] | None = None,
+        reject_output: str
```

Comment:
This is a pretty long type that's hard to read. You can define an alias for this at the top of the file and use it for both parameters `str | re.Pattern[str] | Iterable[str | re.Pattern[str]] | None `.

Reply:
Done in 52e0783021d66aabf7e0d9f8e47b13496c51866a.